### PR TITLE
Some mar file updates for frontend

### PR DESCRIPTION
**Pull request by dr-frmr**

### CODE REVIEW

File: lib/zig/indexer.hoon [hunk 1]
```diff
@@ -279,7 +279,7 @@
         ==
       ::  wheat
       :~  [%is-data %b %|]
-          [%cont (numb (jam code.p.item))]
+          [%cont (numb 0)]
```
Comment (dr-frmr) [inline lib/zig/indexer.hoon:282]: Note @hosted-fornet this process was causing serious delay when looking at a contract on frontend. We should come up with an alternative mechanism to display the contract's nock code, preferably pretty-printed, preferably once (given that deployed contracts will likely be immutable)

Comment (hosted-fornet) [reply lib/zig/indexer.hoon:282]: Did you confirm this speeds up the page spinner? I had tracked it down to a different source, namely a `/hash` query for the zigs contract, which gave hundreds of batches worth of history of all `/source`d zigs accounts. That said, I agree with this change.

Comment (dr-frmr) [reply lib/zig/indexer.hoon:282]: Yeah, it worked for me

Comment (hosted-fornet) [reply lib/zig/indexer.hoon:282]: Cool :+1:

Comment (hosted-fornet) [reply lib/zig/indexer.hoon:282]: Tested on a fakeship testnet? The thing I was seeing will only be slow with many batches and many users.

Comment (dr-frmr) [reply lib/zig/indexer.hoon:282]: Yes, this just solves slowness of one kind. We still need a slowness solution for the thing you're talking about.

**Pull request merged**